components/tidb_query/src/codec/mysql/time/mod.rs

```diff
@@ -468,6 +468,26 @@ mod parser {
             _ => None,
         }
     }
+
+    pub fn parse_from_u64(
```

Comment:
It's better to add some comment for export method.

Reply:
OK.